test/utils/image/manifest.go

```diff
@@ -125,6 +125,7 @@ var (
 	NoSnatTestProxy          = Config{e2eRegistry, "no-snat-test-proxy", "1.0"}
 	// Pause - when these values are updated, also update cmd/kubelet/app/options/container_runtime.go
 	Pause               = Config{gcRegistry, "pause", "3.1"}
+	Perl                = Config{dockerLibraryRegistry, "perl", "5.26"}
```

Comment:
nit: remove this (unless you plan to add the test back in this PR)

Reply:
Fixed (and image_list.go).